pyproject.toml

```diff
@@ -175,6 +175,7 @@ omit = [
     "tests/*",
     "samples/*",
     "singer_sdk/helpers/_compat.py",
+    "singer_sdk/testing/*",
```

Comment:
Why ignore these for coverage?

Reply:
I didn't think it made sense to cover them as I can't see a way to easily test the test framework (as it relies on the pytest plugin) 🤔 I suppose we can test the individual tests, so I can roll back the ignore and create an issue for "testing the test cases". WDYT?

Reply:
Yeah, I think that's what we want eventually to be safe

Reply:
Done ✅